test/controllers/feeds_controller_test.rb

```diff
@@ -23,25 +20,20 @@ class FeedsControllerTest < ActionDispatch::IntegrationTest
   # end
 
   test "should show feed" do
-    get feed_url(@feed)
-    assert_response :success
-  end
-
-  test "should get edit" do
-    get edit_feed_url(@feed)
+    get podcast_feed_url(podcast, feed)
     assert_response :success
   end
 
   test "should update feed" do
-    patch feed_url(@feed), params: {feed: {}}
-    assert_redirected_to feed_url(@feed)
+    patch podcast_feed_url(podcast, feed), params: {feed: {}}
```

Comment:
I'd probably try setting a couple of actual params here.

And should also add tests for:

- Authorizes `#show`
- Authorizes `#update`
- Validates `#update`